library/core/src/lib.rs

```diff
@@ -104,6 +104,8 @@
 // Do not check link redundancy on bootstraping phase
 #![allow(rustdoc::redundant_explicit_links)]
 #![warn(rustdoc::unescaped_backticks)]
+// FIXME: Remove after `no_sanitize` stabilization (along with `#![feature(no_sanitize)]`)
+#![allow(stable_features)]
```

Comment:
You can change the `#![feature(no_sanitize)]` below to `#![cfg_attr(bootstrap, feature(no_sanitize))]` and that will also ensure it is removed once a compiler with the feature stabilized is used for bootstrapping 🙂

Reply:
Thanks for pointing this out! I didn't know this. Done.